yesod-websockets/chat-with-multiple-channels.hs

```diff
@@ -15,12 +15,12 @@ import UnliftIO.Exception (try, SomeException)
 
 data App = App (TVar (M.Map Text (TChan Text, Int)))
 
-instance Yesod App
-
 mkYesod "App" [parseRoutes|
 / HomeR GET
 |]
 
+instance Yesod App
```

Comment:
As I understand, there was a change in Template Haskell in recent GHC versions that made this change necessary. I don't know the details, sorry.